sample/sample-app/build.gradle

```diff
@@ -5,6 +5,10 @@ plugins {
     id 'kotlin-android'
 }
 
+apply plugin: "com.dropbox.affectedtasksplugin"
+
+TestTasks.variantToTest = "debug"
```

Comment:
I envisioned this being in the `affectedModuleDetector` block.  What do you think?